dbus/src/channel/ffichannel.rs

```diff
@@ -284,8 +284,14 @@ impl Channel {
     /// Removes a message from the incoming queue, or waits until timeout if the queue is empty.
     ///
     pub fn blocking_pop_message(&self, timeout: Duration) -> Result<Option<Message>, Error> {
+        self.blocking_pop_message_with_optional_timeut(Some(timeout))
+    }
+
+    /// Removes a message from the incoming queue, or waits until timeout if the queue is empty. If timeout is None, it will wait forever.
+    ///
+    pub fn blocking_pop_message_with_optional_timeut(&self, timeout: Option<Duration>) -> Result<Option<Message>, Error> {
```

Comment:
You also need to modify `nativechannel::Channel`.
Either `ffichannel::Channel` or `nativechannel::Channel` is chosen, depending on the feature flag [here](https://github.com/diwic/dbus-rs/blob/master/dbus/src/channel.rs#L8-L16).